cmplibrary/src/main/java/com/sourcepoint/cmplibrary/unity3d/UnityUtils.kt

```diff
@@ -9,3 +9,7 @@ fun throwableToException(throwable: Throwable) {
 fun <T> arrayToList(array: Array<T>): List<T> {
     return array.toList()
 }
+
+fun <T> arrayToSet(array: Array<T>): Set<T> {
+    return array.toSet()
+}
```

Comment:
@Nevazhnovu there's a chance the build is failing because of klint. I _think_ that missing new line in the end of the file might be the cause.